modular_skyrat/code/modules/job/job_types/chaplain.dm

```diff
@@ -1,4 +1,4 @@
-/datum/job/chaplain
+/datum/outfit/job/chaplain
 	implants = list(/obj/item/implant/sad_trombone/mortis)
 
 /obj/item/implant/sad_trombone/mortis
```

Comment:
Why is this a change in telescience PR?

Reply:
because for some reason it wasn't changed already in master
and it cause compile errors